BedrockServer.cpp

```diff
@@ -1677,6 +1677,7 @@ void BedrockServer::_status(unique_ptr<BedrockCommand>& command) {
         content["host"] = args["-nodeHost"];
         content["commandCount"] = BedrockCommand::getCommandCount();
         content["isDetached"] = isDetached() ? "true" : "false";
+        content["outstandingFramesToCheckpoint"] = _syncNode->getOutstandingFramesToCheckpoint();
```

Comment:
pretty sure you need to use a copy of the sync node like this:

```cpp
    auto _syncNodeCopy = atomic_load(&_syncNode);
    if (_syncNodeCopy) {
        content["outstandingFramesToCheckpoint"] = _syncNodeCopy->getOutstandingFramesToCheckpoint();
    }
```

to avoid race conditions because these commands can run in any thread, not just the sync thread.

Reply:
The reason this is required is because `_syncNode` can get deleted when the sync thread ends at shutdown of for backups, this forces it to continue existing even if the sync thread finishes, until we're done with it here as well.